cloudbuild.yaml

```diff
@@ -110,19 +140,25 @@ steps:
 #
 
 - name: "make-docker"
-  id: lint
+  id: lint-amd64
```

Comment:
Not sure why the arch specific items here? Go Linting isn't architecture specific.

Reply:
Will drop